environments/icds-cas/postgresql.yml

```diff
@@ -0,0 +1,113 @@
+DEFAULT_POSTGRESQL_HOST: pgmain0
+REPORTING_DATABASES:
+  ucr: ucr
+  icds-ucr:
+    WRITE: icds-ucr
+    READ:
+      - [icds-ucr, 1]
```

Comment:
I don't want to remove the main UCR DB from reads completely since our query code will fall back to it if the replication delay on the standby's gets too high but I've increased the weighting of the standbys: 3f3330a